trade-finance-manager-ui/templates/_partials/primary-navigation.njk

```diff
@@ -4,22 +4,30 @@
   {{ mojPrimaryNavigation({
     label: "Primary navigation",
     items: [
+      {
+        text: "Bank reports",
+        href: "/utilisation-reports",
+        active: (activePrimaryNavigation === 'utilisation reports'),
+        attributes: {
+          "data-cy": "utilisation-reports-link"
+        }
+      },
       {
         text: "All deals",
         href: "/deals",
         active: (activePrimaryNavigation === 'all deals'),
         attributes: {
           "data-cy": "all-deals-link"
         }
-    },
-    {
+      },
+      {
         text: "All facilities",
         href: "/facilities",
         active: (activePrimaryNavigation === 'all facilities'),
         attributes: {
           "data-cy": "all-facilities-link"
         }
-    }
+      }
     ]
   }) }}
 {% endif %}
```

Comment:
Just in case anyone comments on these changes - I'm updating it all in FN-1404 (specifically only rendering certain navItems depending on the user team)